mmv1/third_party/terraform/tests/data_source_dns_managed_zone_test.go.erb

```diff
@@ -15,24 +16,32 @@ func TestAccDataSourceDnsManagedZone_basic(t *testing.T) {
 
 	vcrTest(t, resource.TestCase{
 		PreCheck:     func() { testAccPreCheck(t) },
-		Providers:    testAccProviders,
-		CheckDestroy: testAccCheckDNSManagedZoneDestroyProducer(t),
+		ProtoV5ProviderFactories: map[string]func() (tfprotov5.ProviderServer, error){
+			"google": func() (tfprotov5.ProviderServer, error) {
+				provider, err := MuxedProviders(t.Name())
+				return provider(), err
+			},
+		},
+		CheckDestroy: testAccCheckDNSManagedZoneDestroyProducerFramework(t),
 		Steps: []resource.TestStep{
 			{
 				Config: testAccDataSourceDnsManagedZone_basic(randString(t, 10)),
 				Check: checkDataSourceStateMatchesResourceStateWithIgnores(
 					"data.google_dns_managed_zone.qa",
 					"google_dns_managed_zone.foo",
 					map[string]struct{}{
```

Comment:
The acceptance test changes that compare the new and old providers were eaten by rebasing 😢  - they're needed to approve this PR

Reply:
ah!! 🤦‍♀️  - i figured this was going to happen. my bad, will get those back

Reply:
I checked - [the acc test commit](https://github.com/GoogleCloudPlatform/magic-modules/pull/7182/commits/ae6a45996cb26ac6e1c0dffd0e5a0cf58e900197) also included the test helper functions, which I think you subsequently put in a framework test utils file

Reply:
I'm not sure if it's strictly needed, but the original acceptance test PR into this PR included comparing values between the two test runs (see use of `dns_name1` etc[ in the diff here](https://github.com/hashicorp/magic-modules/pull/7/files)). Do you think this helps improve the test? For the managed zone resource it's unclear what field(s) could differ.

Reply:
We decided it's not needed for this data source